.github/workflows/build-cfs-rtems4.11.yml

```diff
@@ -0,0 +1,123 @@
+name: Build and Test rtems 4.11 [OMIT_DEPRECATED=true]
+
+# Run every time a new commit pushed or for pull requests
+on:
+  push:
+  pull_request:
+
+env:
+  OMIT_DEPRECATED: true
+
+jobs:
+  #Checks for duplicate actions. Skips push actions if there is a matching or duplicate pull-request action. 
+  check-for-duplicates:
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+        should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          concurrent_skipping: 'same_content'
+          skip_after_successful_duplicate: 'true'
+          do_not_skip: '["pull_request", "workflow_dispatch", "schedule"]'
+          
+  build-cfs:
+    #Continue if check-for-duplicates found no duplicates. Always runs for pull-requests.
+    needs: check-for-duplicates
+    if: ${{ needs.check-for-duplicates.outputs.should_skip != 'true' }}
+    name: Build
+    runs-on: ubuntu-18.04
+    container: nmullane/qemu_rtems:4.11
+
+    strategy:
+      fail-fast: false
+      matrix:
+        buildtype: [debug, release]
+
+    # Set the type of machine to run on
+    env:
+      BUILDTYPE: ${{ matrix.buildtype }}
+      # Set home to where rtems is located
+      HOME: /root
+
+    steps:
+      # Check out the cfs bundle
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      # Setup the build system
+      - name: Copy Files
+        run: |
+          cp ./cfe/cmake/Makefile.sample Makefile
+          cp -r ./cfe/cmake/sample_defs sample_defs
+
+      # Setup the build system
+      - name: Make Prep
+        run: make SIMULATION=i686-rtems4.11 prep
+
+      - name: Make
+        run: make
+
+  test-cfs:
+    name: Test
+    runs-on: ubuntu-18.04
+    container: nmullane/qemu_rtems:4.11
+
+    needs: build-cfs
+
+    strategy:
+      fail-fast: false
+      matrix:
+        buildtype: [debug, release]
+
+    # Set the type of machine to run on
+    env:
+      BUILDTYPE: ${{ matrix.buildtype }}
+      ENABLE_UNIT_TESTS: true
+      # Set home to where rtems is located
+      HOME: /root
+
+
+    steps:
+      # Checks out a copy of your repository on the ubuntu-latest machine
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      # Setup the build system
+      - name: Copy Files
+        run: |
+          cp ./cfe/cmake/Makefile.sample Makefile
+          cp -r ./cfe/cmake/sample_defs sample_defs
+          # Make sure we are building for the correct target
+          echo "SET(cpu1_SYSTEM i686-rtems4.11)" >> sample_defs/targets.cmake
```

Comment:
This can be removed like it was in other places.

Reply:
:+1: Resolved